alchemist-physics/src/main/kotlin/it/unibo/alchemist/model/implementations/reactions/Physical.kt

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (C) 2010-2022, Danilo Pianini and contributors
+ * listed, for each module, in the respective subproject's build.gradle.kts file.
+ *
+ * This file is part of Alchemist, and is distributed under the terms of the
+ * GNU General Public License, with a linking exception,
+ * as described in the file LICENSE in the Alchemist distribution's top directory.
+ */
+
+package it.unibo.alchemist.model.implementations.reactions
+
+import it.unibo.alchemist.model.implementations.actions.CognitiveAgentCombineSteering
+import it.unibo.alchemist.model.implementations.actions.steeringstrategies.DistanceWeighted
+import it.unibo.alchemist.model.implementations.positions.Euclidean2DPosition
+import it.unibo.alchemist.model.interfaces.Environment
+import it.unibo.alchemist.model.interfaces.Node
+import it.unibo.alchemist.model.interfaces.SteeringAction
+import it.unibo.alchemist.model.interfaces.Time
+import it.unibo.alchemist.model.interfaces.TimeDistribution
+import it.unibo.alchemist.model.interfaces.environments.Dynamics2DEnvironment
+
+/**
+ * A reaction for the update of a [Dynamics2DEnvironment].
```

Comment:
This needs **way** more explanation :smiley: 
What is the idea?

Reply:
In particular, why a reaction and not an action?

Reply:
To update the internal `world` I need to know how much time has elapsed. I figured out I can obtain that information from the reaction level. 
Also, I began writing it following what was done for the now removed `BlendedSteeringWithPhysics`. I needed the `steeringStrategy` computation to obtain the next step position, taking into consideration the physical forces.

Reply:
ok